src/domain/fx/index.js

```diff
@@ -0,0 +1,107 @@
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Vijay Kumar Guthi <vijaya.guthi@infitx.com>
+ --------------
+ ******/
+
+'use strict'
+
+/**
+ * @module src/domain/transfer/
+ */
+
+const ErrorHandler = require('@mojaloop/central-services-error-handling')
+const Metrics = require('@mojaloop/central-services-metrics')
+const FxTransferModel = require('../../models/fxTransfer')
+// const TransferObjectTransform = require('./transform')
+const Cyril = require('./cyril')
+
+const handleFulfilResponse = async (transferId, payload, action, fspiopError) => {
+  const histTimerTransferServiceHandlePayeeResponseEnd = Metrics.getHistogram(
+    'fx_domain_transfer',
+    'prepare - Metrics for fx transfer domain',
+    ['success', 'funcName']
+  ).startTimer()
+
+  try {
+    await FxTransferModel.fxTransfer.saveFxFulfilResponse(transferId, payload, action, fspiopError)
+    // TODO: Need to return a result if we need
+    // const result = TransferObjectTransform.toTransfer(fxTransfer)
+    const result = {}
+    histTimerTransferServiceHandlePayeeResponseEnd({ success: true, funcName: 'handleFulfilResponse' })
+    return result
+  } catch (err) {
+    histTimerTransferServiceHandlePayeeResponseEnd({ success: false, funcName: 'handleFulfilResponse' })
+    throw ErrorHandler.Factory.reformatFSPIOPError(err)
+  }
+}
+
+const forwardedFxPrepare = async (commitRequestId) => {
+  const histTimerTransferServicePrepareEnd = Metrics.getHistogram(
+    'fx_domain_transfer',
+    'prepare - Metrics for fx transfer domain',
+    ['success', 'funcName']
+  ).startTimer()
+  try {
+    const result = await FxTransferModel.fxTransfer.updateFxPrepareReservedForwarded(commitRequestId)
+    histTimerTransferServicePrepareEnd({ success: true, funcName: 'forwardedFxPrepare' })
+    return result
+  } catch (err) {
+    histTimerTransferServicePrepareEnd({ success: false, funcName: 'forwardedFxPrepare' })
+    throw ErrorHandler.Factory.reformatFSPIOPError(err)
+  }
+}
+
+// TODO: Need to implement this for fxTransferError
```

Comment:
Is this implementation still needed?

Reply:
@vijayg10 Can this be removed/moved to a story? Not sure of context.
Going off of git blame for the comment.

Reply:
Created a follow up story. Removed.